search/typo_suggest_engine.go

```diff
@@ -13,9 +14,43 @@ import (
 	null "gopkg.in/volatiletech/null.v6"
 )
 
+type ConstantTerms struct {
+	pattern string
+	terms   []string
+}
+
+func (c *ConstantTerms) RememberTerms(s string) string {
+	if len(c.pattern) > 0 {
+		t := regexp.MustCompile(c.pattern).FindAllStringSubmatch(s, -1)
```

Comment:
Better to use 'Compile' here and not 'MustCompile' to avoid panics in case a bad pattern will be assigned from outside. 
Or add *Regexp object field in ConstantTerms struct instead of pattern string to avoid potential errors on pattern assignment.